clab/docker.go

```diff
@@ -179,6 +187,45 @@ func (c *cLab) CreateContainer(ctx context.Context, node *Node) (err error) {
 	return createContainerNS(node.Pid, node.LongName)
 }
 
+func (c *cLab) PullImageIfRequired(ctx context.Context, node *Node) (err error) {
+	filter := filters.NewArgs()
+	filter.Add("reference", node.Image)
+
+	ilo := types.ImageListOptions{
+		All:     false,
+		Filters: filter,
+	}
+
+	log.Debugf("Looking up %s Docker image", node.Image)
+
+	images, err := c.DockerClient.ImageList(ctx, ilo)
+	_ = images
```

Comment:
why is this needed?

Reply:
L202 is not needed any more, you're right I'm now using that value.